src/jira/client/index.ts

```diff
@@ -43,7 +43,7 @@ async function getJiraClient(
 
 	// TODO: need to create actual class for this
 	const client = {
-		baseURL: instance.defaults.baseURL,
+		baseURL: installation.jiraHost,
```

Comment:
Is it safe to do this change?

Reply:
Yeah, I noticed today in the logs that in some cases `instance.defaults.baseURL` is undefined which isn't good.  `installation.jiraHost` is used a few lines up to set the BaseURL for the axios instance.